Makefile

```diff
@@ -218,6 +218,15 @@ generate/kuma-cp/migrations:
 generate/gui: ## Generate gGOFLAGSo files with GUI static files to embed it into binary
 	GOFLAGS='${GOFLAGS}' go generate ./app/kuma-ui/pkg/resources/...
 
+generate/envoy-imports:
+	echo 'package envoy\n' > imports.go
+	echo '// Import all Envoy packages so protobuf are registered and are ready to used in functions such as MarshalAny.' >> imports.go
+	echo '// This file is autogenerated. run "make generate/envoy-imports" to regenerate it after go-control-plane upgrade' >> imports.go
+	echo 'import (' >> imports.go
+	go list github.com/envoyproxy/go-control-plane/... | grep "github.com/envoyproxy/go-control-plane/envoy/" | awk '{printf "\t_ \"%s\"\n", $$1}' >> imports.go
```

Comment:
Can we run into versioning issues here? Is there a way to  `pin`  what we test and support?

Reply:
As far as I know it uses version from `go.mod`. The only problem here is that after upgrade go-control-plane we need to rerun it, because there might be new packages.